src/Dapr.AspNetCore/DaprBulkAppResponseStatus.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2021 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.AspNetCore
+{
+        /// <summary>
+        /// Status of the message handled in bulk subscribe handler.
+        /// </summary>
+        public enum DaprBulkAppResponseStatus {
```

Comment:
ideally this enum can be used for normal subscribe response to, if there is a domain model for normal subscribe response ...

Reply:
We don't have that yet.